documents/documentContentService.go

```diff
@@ -0,0 +1,53 @@
+package documents
+
+import (
+	"context"
+
+	"github.com/vireocloud/property-pros-service/interfaces"
+	"github.com/vireocloud/property-pros-service/interop"
+)
+
+type DocumentContentService struct {
+	notePurchaseAgreementServiceDocClient interop.NotePurchaseAgreementServiceClient
+	uploader                              interfaces.IDocUploader
+}
+
+func (docs *DocumentContentService) BuildNotePurchaseAgreement(ctx context.Context, payload *interop.NotePurchaseAgreement) (interfaces.IDocumentContent, error) {
+	// TODO: Currently the client errs for unknown reasons, will wait to uncomment it once the doc-service is fixed.
```

Comment:
Currently, the code for the client is commented out, as the client returns errors. Once the client gets fixed, all the TODOs can be removed.